frontend/src/services/graphql/Datasets/index.js

```diff
@@ -0,0 +1,8 @@
+export * from './addDatasetStorageLocation';
```

Comment:
All imports for Folder's view should be fixed according to a new location of services